pkgs/by-name/de/deno/package.nix

```diff
@@ -25,12 +31,20 @@ rustPlatform.buildRustPackage rec {
     owner = "denoland";
     repo = "deno";
     tag = "v${version}";
-    hash = "sha256-WXAUsBC3nAJcuUB753dpM/WDzqWu+e/Kt/BrwQkk/dY=";
+    hash = "sha256-Ir/jBmXsiM9T3dY6kd8wsC+qB1alHm5JNAx+aq9hXHo=";
+    fetchSubmodules = true; # required for tests
   };
 
   useFetchCargoVendor = true;
   cargoHash = "sha256-t44Q4yBdcYAk6jkRrzAHXBsJTsRTHAD95Wyxd3waaHc=";
 
+  patches = [
+    ./patches/unvendor-sqlite.patch
+    ./patches/tests-no-chown.patch
+    ./patches/tests-replace-hardcoded-paths.patch
+    ./patches/tests-darwin-differences.patch
```

Comment:
```suggestion
    ./unvendor-sqlite.patch
    ./tests-no-chown.patch
    ./tests-replace-hardcoded-paths.patch
    ./tests-darwin-differences.patch
```

Reply:
I'm not sure I follow, do you suggest moving the patch files out of the subdirectory?

Reply:
Yep